robosuite_env.py

```diff
@@ -378,23 +428,25 @@ def py3main():
             policy = LatentMotorPlansAlgo(config)
             policy._prepare_for_test_rollouts(args.policy)
 
+            env = RealSawyerLift(control_freq=args.ctrl_freq, use_image=args.image, use_proprio=args.proprio, joint_proprio=False)
+
             obs_seq, _, _, _, _, _, _ = policy.memory.get_trajectory_at_index(-1)
             goal_inds = list(range(len(obs_seq)))[policy.config.train.seq_length :: policy.config.train.seq_length]
             goal_inds.append(-1)
-            env = RealSawyerLift(control_freq=args.ctrl_freq)
 
         else:
-            env = RCANRealSawyerLift(restore_rcan, rcan_kwargs, env_kwargs)
+            env = RCANRealSawyerLift(restore_rcan, rcan_kwargs, env_kwargs={'use_image':args.image,
```

Comment:
Are these args required for penv and py36 both? Do they need to be the same for both? Otherwise, let's try to just call the args where it matters